### ISSUE TITLE
Preferable version of the original game

### ISSUE THREAD

**Issue by Branikolog** [ihhub/fheroes2#4002]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

There's an issue with video files in dosbox version of the original game.
Installation guide lacks the information, that win version is required for fheroes2 engine.

### Describe the possible solution

We should note, that windows version of the original game is preferable for fheroes2 installation.

**Comment by oleg-derevenetz** [comment]
Hi @Branikolog what is the issue exactly? Video files are missing, or maybe engine is unable to play them, or something else?

**Comment by ihhub** [comment]
Hi @oleg-derevenetz , video files are a part of gog file which is essentially ISO. We don't have code to read this file so DOS version of the game doesn't have campaign videos. I made some research in this ground and found few libraries but my main machine is down for a while :(

**Comment by undef21** [comment]
@Branikolog, dosbox version contain homm2.gog file which is raw cd image.
User have to copy heroes2/anim directory to fheroes2 dir.
He can:
1.  rename this file to .img and mount this one with cd emulation software (eg DaemonTools, Alcohol120%, AcetoneISO etc)
or
2. convert this file to .iso for mounting or extracting with archivers.

This is the only way to get localized video now because all non-english gog versions are DOS versions.

**Comment by oleg-derevenetz** [comment]
Unfortunately, Windows by itself isn't able to mount `homm2.gog` file (and `Mount-DiskImage` from PS naturally isn't able to do this either). However, DOSBOX (that is bundled with HoMM2 distribution) is able to do this via its `IMGMOUNT` command. So it is possible to create a helper script that will extract these animations (and even make this a sort of user-friendly). However, I'm not sure about Linux/macOS users in this regard - they will need either to install macOS or use some 3rd party tools to mount VCDs.

**Comment by ihhub** [comment]
Hi @oleg-derevenetz , we can proceed with a script for Windows users (70% of downloads are from them). How are we going to position this script? Under which circumstances?

**Comment by link2xt** [comment]
On Linux I successfully extracted .gog with
```
mkdir -p ANIM && dosbox . -c "imgmount D homm2.gog -t iso -fs iso" -c "mount E ANIM" -c "copy D:\HEROES2\ANIM\*.* E:"
```

@oleg-derevenetz 
I tried to use `homm2.ins` instead of `homm2.gog` on Linux and it didn't work with DOSBox 0.74, shouldn't this line use `homm2.gog` instead of `homm2.ins` too?
https://github.com/ihhub/fheroes2/blob/df644f7d55764b0a04929b7592fe20fb16039c80/script/homm2/extract_homm2_dos_anim.ps1#L18

**Comment by oleg-derevenetz** [comment]
@link2xt

> I tried to use homm2.ins instead of homm2.gog on Linux and it didn't work with DOSBox 0.74, shouldn't this line use homm2.gog instead of homm2.ins too?

To use `homm2.ins`, you should have not only it and `homm2.gog`, but also `MUSIC` directory, because `homm2.ins` mounts CD soundtracks from this directory. This perfectly works with DOSBox 0.74 on Windows. However, if you take a closer look, you will notice that in the `homm2.ins` the `MUSIC` directory goes in the lowercase:

```
FILE "music\homm2_01.ogg" MP3
  TRACK 02 AUDIO
      INDEX 01 00:00:00
```

That's why it may not work on Linux, I suppose.

**Comment by oleg-derevenetz** [comment]
@link2xt

> We don't need to mount the music anyway as it is already extracted.

Yes, of course. I just trying to guess why mounting the `homm2.ins` didn't work on Linux.

> Just saying mounting .gog directly works for me on Linux.

I suppose we still need to install DOSBox first (from some package)? On Windows it's a bit easier, because DOSBox goes with the original game itself, so there is no need for the end user to install it separately.

**Comment by link2xt** [comment]
> I suppose we still need to install DOSBox first (from some package)? 

Yes, just run `command -v dosbox` and ask the user to install dosbox if it's missing, should be ok.

**Comment by oleg-derevenetz** [comment]
I suppose this may be closed due to the #4009 and #4652.
